recipe/meta.yaml

```diff
@@ -23,21 +24,21 @@ requirements:
   host:
     - cymem >=2.0.2,<2.1.0
     - cython >=0.25,<3.0
-    - cython-blis >=0.7.8,<0.8.0
+    - cython-blis >=1.0.0,<1.1.0
     - murmurhash >=1.0.2,<1.1.0
-    - numpy {{ numpy }}
+    - numpy >=2.0.0,<2.1.0
     - pip
     - preshed >=3.0.2,<3.1.0
     - python
     - setuptools
     - wheel
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=2.0.0,<2.1.0
```

Comment:
you can do this by using pin_compatible's min_pin argument, which I think is more maintainable and removes code redundancy